### ISSUE TITLE
[Feature Request] Catch-all signal handler

### ISSUE THREAD

**Issue by lorensr** [temporalio/sdk-typescript#846]
### Is your feature request related to a problem? Please describe.

With multiple handlers, each handler processes its own signals in order. In order to get in-order delivery of all signals, you need to either:

- only send a single signal type (["fat handler"](https://docs.temporal.io/typescript/workflows#define-signals-and-queries-dynamically))
- have a catch-all handler that is called when any signal arrives, regardless of type

### Describe the solution you'd like

Something like:

```ts
import { setGlobalSignalHandler } from '@temporalio/workflow'

setGlobalSignalHandler((signalType: string, arg1: Type1, arg2: Type2) => ...)
```

When both a global and individual handler is defined, do we only deliver to one of them, or to both?

**Comment by bergundy** [comment]
Easy to add, has been brought up before, API TBD.

For the behavior, I'd recommend this being a catch all, meaning it doesn't get triggered if a more specific handler is set.
